src/components/__mocks__/Countries.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+export const fetchCountriesFromApi = () => fetch('https://restcountries.com/v3.1/name/kenya').then((resp) => {
+  if (resp.status === 200) return resp.json();
+  throw new Error('Invalid Response');
+});
+
+const Countries = () => {
+  const [countries, setCountries] = useState([]);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    fetchCountriesFromApi()
+      .then((data) => setCountries(data.results))
+      .catch(() => setError(true));
+  }, []);
+
+  return error ? (
+    <div data-testid="fetch-error">Unable To fetch data</div>
+  ) : (
+    <div data-testid="fetch-success">
+      {countries.map((country) => (
+        <h2 key={country.id}>
+          {country.cca2}
+        </h2>
+      ))}
+    </div>
+  );
+};
+
+export default Countries;
```

Comment:
- Hey, good job so far! Currently, your Homepage is having one column instead of two below this is a result of the `width: 45%` you added in the `App.css` ` .country-card {`. kindly remove or adjust the styling to match the design template.


![Screenshot from 2022-10-15 13-10-35](https://user-images.githubusercontent.com/59397159/195985916-c34fd68a-77b2-4c11-aea9-6874767b77ba.png)

Reply:
- Also, according to the project requirement below: 
![Screenshot from 2022-10-15 13-01-48](https://user-images.githubusercontent.com/59397159/195986085-7a3ed25d-6b8a-4f04-8c7e-a3ad5f259956.png)

- Kindly add a search bar or something similar to enforce an interaction by which users could search for a country name, instead of scrolling down the page when looking for a particular country to meet this part of the project requirement.

Reply:
Thank you for the feedback @zieeco 
The mentioned issues have been resolved.